examples/integrations/langchain/langchain.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>Small typo : "show" instead of "shows" right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>